components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/internal/model/common/Group.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.webhook.wso2.event.handler.internal.model.common;
+
+import java.util.List;
+
+public class Group {
+
+    private String id;
+    private String name;
+    private List<User> addedUsers;
+    private List<User> removedUsers;
+    private String ref;
+
+    // Getters and setters
```

Comment:
remove this

Reply:
Fixed 27075e7f4f103ad21dd5762d4ee9b01a4ba63bcf